internal/app/update.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package app
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+)
+
+type Release struct {
+	URL     string `json:"html_url"`
+	Version string `json:"name"`
+}
+
+func GetLatestRelease() (Release, error) {
+	var release Release
+
+	ctx := context.Background()
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, "https://api.github.com/repos/sighupio/furyctl/releases/latest", nil)
```

Comment:
it would be nice to have a timeout in the context so as to not hang the command when the user is offline